main.go

```diff
@@ -14,13 +17,40 @@ var (
 
 func main() {
 	app := cli.NewApp()
-	app.Name = "rancher-vxlan"
 	app.Version = VERSION
 	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "listen",
+			Value: ":8111",
+		},
 		cli.BoolFlag{
 			Name:   "debug, d",
 			EnvVar: "RANCHER_DEBUG",
 		},
+		cli.BoolFlag{
+			Name:   "disable-tx-offload",
```

Comment:
Did you test with both enabling/disabling this? I had to disable this inside router otherwise, the packets were getting dropped. Since we are running in hostns, do we even need this now? If this is not needed, can you create a new github issue as an enhancement and we can work on this later. 

If we don't need this, let's get rid of this option all together.